db/migrations/01_create_peeps_table.sql

```diff
@@ -0,0 +1 @@
+CREATE TABLE peeps(id SERIAL PRIMARY KEY, content VARCHAR(250));
```

Comment:
I will suggest a more specific key name instead of 'content', as in the future it will have more content than the message (peep) itself, like the time, author...